lib/membrane/rtp/twcc_sender.ex

```diff
@@ -32,9 +32,25 @@ defmodule Membrane.RTP.TWCCSender do
   end
 
   @impl true
-  def handle_pad_added(pad, _ctx, state) do
+  def handle_pad_added(pad, ctx, state) do
     {queued_actions, other_actions} = Map.pop(state.buffered_actions, pad, [])
 
+    has_other_input_pads? =
+      ctx.pads
+      |> Map.values()
+      |> Enum.count(&(&1.ref != pad && &1.direction == :input))
+      |> then(&(&1 != 0))
```

Comment:
```suggestion
      |> Map.values()
      |> Enum.any?(&(&1.ref != pad and &1.direction == :input))
```

or using only one iteration

```suggestion
      |> Enum.any?(fn
        {ref, %{direction: :input}} when ref != pad -> true
        _else -> false
      end)
```